requirements/dev.in

```diff
@@ -3,9 +3,9 @@ bumpversion
 pip-tools
 
 # Code formatting
-black
-isort
-flake8
+isort==5.0.7
+black==19.10b0
+flake8==3.8.3
```

Comment:
`*.in` files shouldn't contain pinned version. You can do it temporarily to then run `./bin/compile_dependencies.sh`, but after that specify the unpinned versions again in the .in files.

Reply:
I unpinned them in dev.in again